wfe/wfe.go

```diff
@@ -1230,9 +1230,11 @@ func (wfe *WebFrontEndImpl) Authorization(ctx context.Context, logEvent *web.Req
 	id := request.URL.Path
 	authz, err := wfe.SA.GetAuthorization(ctx, id)
 	if err != nil {
-		// TODO(#1199): handle db errors
-		wfe.sendError(response, logEvent, probs.NotFound("Unable to find authorization"), err)
-		return
+		if berrors.Is(err, berrors.NotFound) {
+			notFound()
```

Comment:
`notFound()` is a local function in `wfe.Challenge` (that also has text specific to challenges in the error msg). Referencing it from `Authorization` is breaking CI:
```
# github.com/letsencrypt/boulder/wfe
wfe/wfe.go:1234:4: undefined: notFound
```